homeassistant/components/dlib_face_detect/image_processing.py

```diff
@@ -4,14 +4,13 @@
 
 import face_recognition  # pylint: disable=import-error
 
+# pylint: disable=unused-import
```

Comment:
This is a weird move?

Looks like it has been done twice, see line #14

Reply:
I think I fixed it. Please resolve if you are happy with the change.